WalletWasabi/Blockchain/Analysis/BlockchainAnalyzer.cs

```diff
@@ -38,7 +38,7 @@ public void Analyze(SmartTransaction tx)
 		{
 			AnalyzeWalletInputs(tx, out HashSet<HdPubKey> distinctWalletInputPubKeys, out int newInputAnonset);
 
-			if (inputCount == ownInputCount)
+			if (!tx.IsLikelyCoinjoin())
```

Comment:
?

Reply:
Idea: using the same metrics everywhere.